### DISCUSSION TITLE
Approved but not received benefits yet

### DISCUSSION THREAD

**Discussion by 001Jay** [community/community#135338] [GitHub Education]
### Hi everyone,

Why is it that the certification has not been successful after three days
![微信截图_20240810204705](https://github.com/user-attachments/assets/f51ab510-ae52-46be-86e8-439bfd3ebdc9)

**Comment by davevad93** [comment]
Hi @001Jay ,

The waiting time may vary depending on different factors, such as the volume of requests. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @001Jay was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135145

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
